<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>syDevicesWidget</class>
 <widget class="QWidget" name="syDevicesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>516</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="deviceGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="title">
      <string>Detected Devices</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="QTreeWidget" name="deviceList">
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>List of installed input/output devices and their default profiles.</string>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="uniformRowHeights">
         <bool>false</bool>
        </property>
        <property name="allColumnsShowFocus">
         <bool>false</bool>
        </property>
        <property name="headerHidden">
         <bool>true</bool>
        </property>
        <property name="columnCount">
         <number>2</number>
        </property>
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>*ICON*</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">2</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="relatedDeviceCheckBox">
        <property name="text">
         <string>Show only device related ICC profiles</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="profileAssociationGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>247</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Profiles Associated with Device</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QListWidget" name="profileAssociationList">
        <property name="toolTip">
         <string>Listing of all profiles that are associated with the selected device.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Available DeviceProfiles</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5">
           <item>
            <widget class="QComboBox" name="deviceProfileComboBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>27</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
